docs/design/network-discovery/port-report-FSM.puml

```diff
@@ -8,8 +8,10 @@ INIT --> DOWN : port-down
 
 UP -> DOWN : port-down
 UP : enter / report "become UP"
+UP : enter / send the event to port history
 
 DOWN -> UP : port-up
 DOWN : enter / report "become DOWN"
+DOWN : enter / send the event to port history
 
-@enduml
+@enduml
```

Comment:
minor: removed blank line in the end of file

Reply:
Fixed